downstream/modules/aap-hardening/proc-configure-external-authentication.adoc

```diff
@@ -7,7 +7,7 @@
 
 [role="_abstract"]
 
-As noted in the link:(add external link) external authentication is recommended for user access to the automation controller. After you choose the authentication type that best suits your needs, navigate to menu:Settings[Authentication] in the automation controller UI, click on the relevant link for your authentication back-end, and follow the relevant instructions for link:https://docs.ansible.com/automation-controller/latest/html/administration/configure_tower_in_tower.html#authentication[configuring the authentication] connection.
+As noted in the link:(add internal link) external authentication is recommended for user access to the automation controller. After you choose the authentication type that best suits your needs, navigate to menu:Settings[Authentication] in the automation controller UI, click on the relevant link for your authentication back-end, and follow the relevant instructions for link:https://docs.ansible.com/automation-controller/latest/html/administration/configure_tower_in_tower.html#authentication[configuring the authentication] connection.
```

Comment:
do we need to add an internal link here?

Reply:
Internal link added.